addons/website_slides_survey/views/website_slides_templates_lesson.xml

```diff
@@ -7,17 +7,33 @@
                     <a role="button"
                         class="btn btn-primary btn-lg"
                         t-att-href="'/slides_survey/slide/get_certification_url?slide_id=%s' %(slide.id)">
-                        <i class="fa fa-fw fa-graduation-cap" role="img"/>
-                        <t t-if="slide.channel_id.is_member and slide.channel_id.active">Begin Certification</t>
+                        <i class="fa fa-trophy" role="img"/>
+                        <t t-if="slide.channel_id.is_member and slide.channel_id.active">Start Certification</t>
                         <t t-else="">Test Certification</t>
                     </a>
                 </div>
-                <div t-if="slide.slide_category == 'certification' and channel_progress[slide.id].get('completed')" class="col mt32 mb8 d-flex justify-content-center">
-                    <a role="button" class="btn btn-primary btn-lg" t-att-href="'/survey/%s/get_certification' % slide.survey_id.id">
+                <div t-if="slide.slide_category == 'certification' and channel_progress[slide.id].get('completed')" class="col mt32 mb8 text-center">
+                    <h4 class="mb-3">Congratulations, you passed the Certification!</h4>
+                    <a role="button" class="btn btn-primary btn-lg mb-3" t-att-href="'/survey/%s/get_certification' % slide.survey_id.id">
                         <i class="fa fa-fw fa-trophy" role="img" aria-label="Download certification" title="Download certification"/> Download certification
                     </a>
                 </div>
             </xpath>
+            <xpath expr="//a[hasclass('o_wslides_done_button')][1]" position="after">
+                <a t-elif="not slide_completed and slide.slide_category == 'certification'"
+                    role="button"
+                    t-attf-class="o_wslides_done_button btn btn-primary border text-white mr-2"
+                    href="#quiz_container">
+                    Take Quiz
```

Comment:
So that is for the case where you have a Certification AND a quiz?
Doesn't the base use case not already handles this?

Not sure I understand.